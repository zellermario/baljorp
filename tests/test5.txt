create L m 0
create Eszkimo e m
pass e