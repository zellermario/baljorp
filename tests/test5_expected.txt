Fields:
L m (0) - object: x
	neighbours: []

Buildings:

Actors:
> Eszkimo e (heat: 5) on m
  Inventory: []